game/static/game/css/board.css

```
@@ -1450,23 +1457,18 @@ body {
-    box-shadow: inset 0 0 8px rgba(220, 38, 38, 0.35), 
-                0 0 12px rgba(220, 38, 38, 0.25);
+    background: rgba(220, 38, 38, 0.18) !important;
+    animation: pulse-check 1s ease-in-out infinite !important;
 }
 
 @keyframes pulse-check {
     0%, 100% {
-        box-shadow: inset 0 0 8px rgba(220, 38, 38, 0.35), 
-                    0 0 12px rgba(220, 38, 38, 0.25);
+        box-shadow: 0 0 0 2px rgba(220, 38, 38, 0.85) inset;
+        background: rgba(220, 38, 38, 0.15) !important;
```

Stylelint (17.12.0)

[error] Disallowed !important (keyframe-declaration-no-important)

(keyframe-declaration-no-important)